### PR TITLE
feat(toolkit-cleaner): allow only manual runs

### PR DESCRIPTION
**Pull request by blimmer**
Thank you for writing this toolkit clean-up logic. It's a great solution & I'd love to see it integration into the core CDK project at some point. This will really help out several of my larger clients that rely heavily on CDK.

In the meantime, I tested this out against my personal stacks and it worked flawlessly. However, I'd like to be able to only run it manually (e.g., not on a schedule) for this account. This PR adds the ability to pass `schedule: null`, which skips creating the eventbridge rule.

### CODE REVIEW

File: src/toolkit-cleaner/index.ts [hunk 1]
```diff
@@ -17,11 +17,13 @@ import { GetStackNamesFunction } from './get-stack-names-function';
...
   /**
-   * The schedule for the cleaner
+   * The schedule for the cleaner.
+   *
+   * Pass `null` to only run manually via the Step Functions interface
    *
    * @default - every day
    */
-  readonly schedule?: Schedule;
+  readonly schedule?: Schedule | null;
```
Comment (jogold) [inline src/toolkit-cleaner/index.ts:26]: Unfortunately we cannot do this kind of union types with `jsii` (see failing build).

Let's add a `enabled` prop and pass it to the `Rule` (which already defaults to `true` for this prop).

**Review by jogold — CHANGES_REQUESTED** [review]

File: src/toolkit-cleaner/index.ts (hunk 1 above)
Comment (blimmer) [reply src/toolkit-cleaner/index.ts]: Sounds good. Updates made!

File: src/toolkit-cleaner/index.ts [hunk 2]
```diff
@@ -17,12 +17,20 @@ import { GetStackNamesFunction } from './get-stack-names-function';
...
    * @default - every day
    */
   readonly schedule?: Schedule;
 
+  /**
+   * Is the schedule active? If you'd like to run the cleanup manually via the
+   * UI, set to `false`.
+   *
+   * @default - true
+   */
```
Comment (jogold) [inline src/toolkit-cleaner/index.ts:31]:
```suggestion
  /**
   * Whether to clean on schedule. If you'd like to run the cleanup manually 
   * via the console, set to `false`.
   *
   * @default true
   */
```

**Review by jogold — APPROVED** [review]
Thanks @blimmer

**Pull request merged**